### PR TITLE
Support notification change account Solana for DApp

### PR DESCRIPTION
**Pull request by hoanganhtuan95ptit**
PR supports notifying DApps of account Solana changes.

Thank you very much.

### CODE REVIEW

File: src/solana_provider.js [hunk 1]
```diff
@@ -49,6 +49,11 @@ class TrustSolanaWeb3Provider extends BaseProvider {
     this.isConnected = true;
   }
 
+  emitAccountChanged(address) {
+
```
Comment (y3fers0n) [inline src/solana_provider.js:53]: Please remove this line

File: src/solana_provider.js [hunk 2]
```diff
@@ -49,6 +49,11 @@ class TrustSolanaWeb3Provider extends BaseProvider {
     this.isConnected = true;
   }
 
+  emitAccountChanged(address) {
```
Comment (y3fers0n) [inline src/solana_provider.js:52]: There is not need to pass address param

**Comment by y3fers0n** [comment]
Thank you for sending this PR 🥳 .
Just a couple of comments and we can approve this

**Review by DSBUGAY2 — COMMENTED** [review]
jpdgE3kWHJfgdTXGCbhBtqe585ofpuyTEGA9g9HCC2h

**Comment by hoanganhtuan95ptit** [comment]
I have updated it according to your instructions, please continue your review. Thank you

File: src/solana_provider.js (hunk 1 above)
Comment (hoanganhtuan95ptit) [reply src/solana_provider.js]: Thank you, I fixed it

File: src/solana_provider.js (hunk 2 above)
Comment (hoanganhtuan95ptit) [reply src/solana_provider.js]: Thank you, I fixed it

**Review by y3fers0n — APPROVED** [review]

File: src/solana_provider.js (hunk 2 above)
Comment (Mohammed-yimam) [reply src/solana_provider.js]: Thanks

**Comment by Mohammed-yimam** [comment]
Thanks god

**Comment by Mohammed-yimam** [comment]
Thanks

**Pull request merged**